static/app/utils/discover/fields.tsx

```diff
@@ -1045,6 +1073,106 @@ export function isNumericMetrics(field: string): boolean {
   ].includes(field);
 }
 
+export const FIELDS_DOCS: Readonly<Record<string, string>> = {
+  has: 'check field exists',
```

Comment:
do we need to localize all these docstrings?
```suggestion
  has: t('check field exists'),
```